File: tests/python/testB.py
class TestB:
    @staticmethod
    def print_hello_test_b1():
        print("Hello, World!")
        for i in range(5):
            print(f"This is line {i}")
            if i % 2 == 0:
                print("Even number")
            else:
                print("Odd number")

    @staticmethod
    def print_hello_test_b2():
        print("Hello, World!")
        for i in range(5):
            print(f"This is line {i}")
            if i % 2 == 0:
                print("Even number")
            else:
                print("Odd number")
        print("End of printHello")

    @staticmethod
    def print_hello_test_b3():
        print("Hello, World!")
        for i in range(5):
            print(f"This is line {i}")
            if i % 2 == 0:
                print("Even number")
            else:
                print("Odd number")

    @staticmethod
    def main():
        TestB.print_hello_test_b1()
        TestB.print_hello_test_b2()
        TestB.print_hello_test_b3()

# Execute main method
TestB.main()

File: tests/java/TestB.java
public class TestB {
    public static void printHelloTestB1() {
        System.out.println("Hello, World!");
        for (int i = 0; i < 5; i++) {
            System.out.println("This is line " + i);
            if (i % 2 == 0) {
                System.out.println("Even number");
            } else {
                System.out.println("Odd number");
            }
        }
    }

    public static void printHelloTestB2() {
        System.out.println("Hello, World!");
        for (int i = 0; i < 5; i++) {
            System.out.println("This is line " + i);
            if (i % 2 == 0) {
                System.out.println("Even number");
            } else {
                System.out.println("Odd number");
            }
        }
        System.out.println("End of printHello");
    }

    public static void printHelloTestB3() {
        System.out.println("Hello, World!");
        for (int i = 0; i < 5; i++) {
            System.out.println("This is line " + i);
            if (i % 2 == 0) {
                System.out.println("Even number");
            } else {
                System.out.println("Odd number");
            }
        }
    }

    public static void main(String[] args) {
        printHello();
        printHelloTestB1();
        printHelloTestB2();
        printHelloTestB3();
    }
}